pressbooks/pressbooks
# fix: render mathjax formulas within glossary terms & tablepress tables

Issue: https://github.com/pressbooks/pressbooks/issues/3408

This PR allows the `latex` shortcode to be rendered in the Glossary Term posts type (web, EPUB and PDF versions) as well as within the glossary term modal in the web version.
In addition, it also allows latex formulas to be properly rendered within TablePress tables.


### Test case
- Create one or more glossary terms using the latex shortcode, for example: `This is a glossary term with a math expression: [latex]x ^(3y) - 9[/latex]`.
- Add 1 or more TablesPress tables with latex formulas in it.
- Use this glossary in one or more chapters using the glossary shortcode. For example, in a post content: `I here's the [glossary id="<YOUR GLOSSARY ID HERE>"]term[/glossary].`.
- Create a Back Matter with `Glossary` Post type. Leave it blank so it can be populated with your list of glossary terms.
- Go to the web version of your posts containing the glossary terms. Clicking on the terms, you should be able to see the glossary pop-up with the mathjax formulas properly rendered as well as the tablepress table with formulas.
- Visit the Back Matter for the list of glossary terms. You should be able to see the mathjax formulas properly rendered for those glossary terms containing mathjax formulas.
- Export the book to PDF and EPUB. You should be able to see the glossary terms with the formulas as expected in the back matter post type.
